client/layout/index.jsx

```diff
@@ -74,6 +77,27 @@ function SidebarScrollSynchronizer() {
 	return null;
 }
 
+function HelpCenterLoader( { sectionName, loadHelpCenter } ) {
+	const { setShowHelpCenter } = useDispatch( HELP_CENTER_STORE );
+	const isDesktop = useBreakpoint( '>760px' );
```

Comment:
This causes a warning in the console `Undefined breakpoint used in `mobile-first-breakpoint` >760px`

<img width="887" alt="Screenshot 2022-09-16 at 10 19 29 AM" src="https://user-images.githubusercontent.com/18226415/190558971-fd3aca69-e4b6-46c2-9071-1c3949e53f87.png">

CC: @Automattic/team-calypso